Firmware/Marlin_main.cpp

```diff
@@ -8609,10 +8633,14 @@ SERIAL_PROTOCOLPGM("\n\n");
             case 1:
                 MMU2::mmu2.Reset(MMU2::MMU2::ResetPin);
                 break;
+            case 42:
+                MMU2::mmu2.Reset(MMU2::MMU2::EraseEEPROM);
+                break;
             default:
                 break;
             }
         }
+        printf_P(_n("MMU state:%d\n"), MMU2::mmu2.Enabled());
```

Comment:
@3d-gussner the PR looks good to me now. But I’m not sure about this log.

I feel it is not descriptive enough.

Maybe `MMU On` and `MMU Off` is better? The. It would be similar to the log I added for SpoolJoin when the printer boots up.